### PR TITLE
Fixed error when start console

### PR DESCRIPTION
**Pull request by rafael-santiago-take**
Fixed error when start console that occured after IFlowLoader implementation

### CODE REVIEW

File: src/Take.Blip.Builder/FlowLoader.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Threading.Tasks;
+using Take.Blip.Builder.Models;
+
+namespace Take.Blip.Builder
+{
+    internal class FlowLoader : IFlowLoader
+    {
+        public Task<Flow> LoadFlowAsync(FlowType flowType, Flow parentFlow, string identifier, CancellationToken cancellationToken)
+        {
+            //Used when occurs a subflow redirect.Wille be implemented in the future.
```
Comment (caiopsouza) [inline src/Take.Blip.Builder/FlowLoader.cs:12]: Suggestion, if I understood correctly what you meant.

`This should be called when redirecting to a subflow. This feature is not supported yet, so we throw an exception.

File: src/Take.Blip.Builder/FlowLoader.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            throw new NotImplementedException();
```
Comment (caiopsouza) [inline src/Take.Blip.Builder/FlowLoader.cs:13]: Could you be more specific with this exception, so we can better track it if it ever occur? Something like: `Loading a subflow from the SDK is not supported.`

**Review by caiopsouza — COMMENTED** [review]
Mostly changes requests to clarify things.

**Review by fadoaglauss — APPROVED** [review]

File: src/Take.Blip.Builder/FlowLoader.cs (hunk 2 above)
Comment (rafael-santiago-take) [reply src/Take.Blip.Builder/FlowLoader.cs]: done

**Review by jujucasseb — APPROVED** [review]

**Pull request merged**